R/open.R

```diff
@@ -0,0 +1,7 @@
+
+open <- function(data = NULL, title = "") {
```

Comment:
so i don't think we can use `open()` as there's already a `base::open()` ... how about `openNew()` ... and not worrying about `newWin() and `newSess()` ... ?

will this work when using "system R"? i feel like it might not.